composer.json

```diff
@@ -5,7 +5,8 @@
     "license": "MIT",
     "require": {
         "php": ">=7.4",
-        "spryker-eco/unzer": "^1.0.0",
+        "spryker-eco/unzer": "dev-feature/cc-16934/dev-unzer-glue-support-integration",
+        "spryker/checkout-rest-api-extension": "^1.3.0",
```

Comment:
* The version "^1.5.0" must be used instead because of the usage  `CheckoutDataExpanderPluginInterface`.

Reply:
Fixed.